### PR TITLE
fix bad taste

### PR DESCRIPTION
**Pull request by qiaolin-li**
## What is the purpose of the change
fix bad taste 

## issue url
https://github.com/apache/dubbo/issues/5919

### CODE REVIEW

**Comment by htynkn** [comment]
@qiaolin-li  thanks for your PR. I just double check this change and notice that the previous code rely on exceptions is not empty. so which means this change is not equivalent to previous code.

**Comment by qiaolin-li** [comment]
I don't understand what you mean..

**Comment by htynkn** [comment]
sorry @qiaolin-li , I think it's my mistake when I first time look at the issue you raised.
I test this change in my local, It's possible if one stream open fail, so exceptions map can be empty, so we can't print `, possible causes:` in error message

File: dubbo-common/src/main/java/org/apache/dubbo/common/extension/ExtensionLoader.java [hunk 1]
```diff
@@ -576,14 +576,10 @@ private IllegalStateException findException(String name) {
             }
         }
         StringBuilder buf = new StringBuilder("No such extension " + type.getName() + " by name " + name);
-
+        buf.append(", possible causes: ");
 
         int i = 1;
         for (Map.Entry<String, IllegalStateException> entry : exceptions.entrySet()) {
```
Comment (htynkn) [inline dubbo-common/src/main/java/org/apache/dubbo/common/extension/ExtensionLoader.java]: if exceptions is empty, we should not print `, possible causes: `

Comment (qiaolin-li) [reply dubbo-common/src/main/java/org/apache/dubbo/common/extension/ExtensionLoader.java]: Yes, let's add a check when exceptions. EntrySet ().size() is greater than 0

Comment (htynkn) [reply dubbo-common/src/main/java/org/apache/dubbo/common/extension/ExtensionLoader.java]: but my idea is we already have `i` in here, so we don't need calculate exceptions size again.
it's more like a trade off, so I would like to keep original code.

Comment (qiaolin-li) [reply dubbo-common/src/main/java/org/apache/dubbo/common/extension/ExtensionLoader.java]: Ok, I see

Comment (qiaolin-li) [reply dubbo-common/src/main/java/org/apache/dubbo/common/extension/ExtensionLoader.java]: Do I close this PR now

Comment (htynkn) [reply dubbo-common/src/main/java/org/apache/dubbo/common/extension/ExtensionLoader.java]: apologize again for the mistake when the first time I check the issue your raised.
I'm going to close this PR. thanks again

**Pull request closed without merging**